wire-library/wire-java-generator/src/test/java/com/squareup/wire/java/JavaGeneratorHelper.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.java
+
+import com.squareup.javapoet.JavaFile
+import com.squareup.wire.java.internal.ProfileParser
+import com.squareup.wire.schema.Location
+import com.squareup.wire.schema.Schema
+import java.io.IOException
+
+/** Helper class to run Java code generation. */
+internal class JavaGeneratorHelper(private val schema: Schema) {
```

Comment:
I generally loathe `Helper` as a naming suffix ’cause it doesn’t define a boundary on what this does. This seems like it’s mostly concerned with profiles?

Maybe JavaWithProfilesGenerator ?